encode/utils.go

```diff
@@ -1,4 +1,4 @@
-package flg
+package encode
 
```

Comment:
These utility functions could be shared across  flag and env or we can move this back into flag.